### ISSUE TITLE
Unable to run node js benchmarks

### ISSUE THREAD

**Issue by nervermore2** [spcl/serverless-benchmarks#161]
Hi I was try to run node js benchmarks using `./sebs.py experiment invoke perf-cost --config config/example.json --deployment aws --verbose --output-dir 120-out-node14/`
Then I got 
```
[19:31:17.171805] AWS.Credentials-f818 Using cached credentials for AWS
[19:31:17.172120] AWS.Resources-80b0 Using cached resources for AWS
[19:31:17.172306] AWS.Config-84d3 Using cached config for AWS
512
ERROR:root:Unsupported nodejs version 14 in aws!
Traceback (most recent call last):
  File "/home/ubuntu/serverless-benchmarks-master/./sebs.py", line 30, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/serverless-benchmarks-master/python-venv/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/serverless-benchmarks-master/python-venv/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/serverless-benchmarks-master/python-venv/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/serverless-benchmarks-master/python-venv/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/serverless-benchmarks-master/python-venv/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/ubuntu/serverless-benchmarks-master/./sebs.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/ubuntu/serverless-benchmarks-master/./sebs.py", line 97, in wrapper
    return func(*args, **kwargs)
  File "/home/ubuntu/serverless-benchmarks-master/./sebs.py", line 463, in experiment_invoke
    experiment.prepare(sebs_client, deployment_client)
  File "/home/ubuntu/serverless-benchmarks-master/sebs/experiments/perf_cost.py", line 49, in prepare
    self._function = deployment_client.get_function(self._benchmark)
  File "/home/ubuntu/serverless-benchmarks-master/sebs/faas/system.py", line 147, in get_function
    raise Exception(
Exception: Unsupported nodejs version 14 in aws!
```

**Comment by nervermore2** [comment]
Did some search on the code, was able to solve the error by updating the config
```
    "runtime": {
      "language": "nodejs",
      "version": "14.x"
    },
``` instead of 
```
    "runtime": {
      "language": "nodejs",
      "version": "14"
    },

**Comment by mcopik** [comment]
@nervermore2 Yes, this is unfortunate mismatch - I will try to fix this.

**Comment by mcopik** [comment]
@nervermore2 This has been fixed in #166 - you can now use the same format for Node.js version numbers everywhere.

Btw, this PR also dropped support for Node.js 12 on AWS - it is no longer possible to create such functions.
